### PR TITLE
More functions from z_actor.c

### PR DESCRIPTION
**Pull request by Cleverking2003**
Decompiled more functions from z_actor.c

### CODE REVIEW

**Review by Kenix3 — COMMENTED** [review]
Looks good to me

File: src/code/z_actor.c [hunk 1]
```diff
@@ -106,11 +135,30 @@ void Actor_TitleCardContextInit(GlobalContext* ctxt, TitleCardContext* titleCtxt
...
 GLOBAL_ASM("asm/non_matchings/z_actor//Actor_Nop800B5E50.asm")
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_TitleCardUpdate.asm")
+void Actor_TitleCardUpdate(GlobalContext* ctxt, TitleCardContext* titleCtxt) {
+    if (DECR(titleCtxt->fadeInDelay) == 0) {
+        if (DECR(titleCtxt->fadeOutDelay) == 0) {
+            Lib_StepTowardsCheck_s(&titleCtxt->alpha, 0, 30);
+            Lib_StepTowardsCheck_s(&titleCtxt->color, 0, 70);
+        }
+        else {
```
Comment (Rozelette) [inline src/code/z_actor.c]: Put the else on the same line as the closing brace.
```suggestion
        } else {
```

File: src/code/z_actor.c [hunk 2]
```diff
@@ -517,7 +582,26 @@ GLOBAL_ASM("asm/non_matchings/z_actor//Actor_InsertIntoTypeList.asm")
...
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_FreeOverlay.asm")
+void Actor_FreeOverlay(ActorOverlayTableEntry* entry) {
+    u32 ramAddr;
+
+    if (entry->clients == 0) {
+        ramAddr = entry->ramAddr;
+        if (ramAddr != 0) {
+	    //Bit 1 - always loaded
+            if ((entry->flags & 2) == 0) {
+		//Bit 0 - don't alloc memory
```
Comment (Rozelette) [inline src/code/z_actor.c]: Looks like some tabs got mixed in. Replace with spaces.
```suggestion
            //Bit 1 - always loaded
            if ((entry->flags & 2) == 0) {
                //Bit 0 - don't alloc memory
```

File: src/code/z_actor.c [hunk 3]
```diff
@@ -517,7 +582,26 @@ GLOBAL_ASM("asm/non_matchings/z_actor//Actor_InsertIntoTypeList.asm")
...
+                if ((entry->flags & 1) != 0) {
+                    entry->ramAddr = 0;
+                }
+                else {
```
Comment (Rozelette) [inline src/code/z_actor.c]: Put the else on the same line as the closing brace.
```suggestion
                } else {
```

**Review by Rozelette — CHANGES_REQUESTED** [review]
Besides minor formatting, looks good!

**Review by Rozelette — APPROVED** [review]

**Pull request merged**